recipes-browser/chromium/chromium-ozone-wayland/0001-Add-support-for-V4L2VDA-on-Linux.patch

```diff
@@ -325,7 +321,7 @@ index 4f87445..9b3c55f 100644
      default:
        DVLOGF(1) << "Unrecognized format " << FourccToString(format);
 diff --git a/media/gpu/v4l2/v4l2_video_decode_accelerator.cc b/media/gpu/v4l2/v4l2_video_decode_accelerator.cc
-index 7e361c3..1d916d1 100644
+index bcaca34dd4a9..bdfe53ada325 100644
 --- a/media/gpu/v4l2/v4l2_video_decode_accelerator.cc
```

Comment:
Is it still necessary to change this file? This extra include was added in 
https://github.com/OSSystems/meta-browser/commit/1f363e25cb7dbbfcd10d96071b133d08fda5cfa2, and https://github.com/OSSystems/meta-browser/commit/37ed2a6062460a1853e768956e50fbe9c5b4530e removed usage of the `BUILDFLAG` macro.

Reply:
What are you referring to? I can't see any changes here except the index and lines because the patch was applied to m75 and formatted back

Reply:
I'm referring to the change to `media/gpu/v4l2/v4l2_video_decode_accelerator.cc` that's part of this patch. It's only adding a header that is not used anywhere in the file.

Reply:
This header? https://github.com/rakuco/meta-browser/pull/1/files/5b320b910eaa125ec0de43a767c695800397f57c#diff-e715e81af9b5a6b33e3c51580809fe26L335

Reply:
Yep.